hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieTimeline.java

```diff
@@ -170,7 +170,7 @@ public interface HoodieTimeline extends Serializable {
    *
    * @return
    */
-  HoodieTimeline filterCompletedInstantsOrRewriteTimeline();
+  HoodieTimeline filterCompletedAndRewriteInstants();
```

Comment:
+1 to `filterCompletedAndRewriteInstants `

Reply:
generally, we have used rewrite to denote only replace commits. So, are we changing the definition ?

Reply:
We are not changing the definition, the instants cover two parts: completed commits and also all the rewrite instants. For rewrite instants, it could include a clustering commit also an insert overwrite.